generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -7045,6 +7045,51 @@ namespace nidaqmx_grpc {
lName_if(channel_names, 53), get_state_if(stateVector, 53), get_channelName_if(channel_names, 54), get_state_if(stateVector, 54), get_channelName_if(channel_names, 55), get_state_if(stateVector, 55), get_channelName_if(channel_names, 56), get_state_if(stateVector, 56), get_channelName_if(channel_names, 57), get_state_if(stateVector, 57), get_channelName_if(channel_names, 58), get_state_if(stateVector, 58), get_channelName_if(channel_names, 59), get_state_if(stateVector, 59), get_channelName_if(channel_names, 60), get_state_if(stateVector, 60), get_channelName_if(channel_names, 61), get_state_if(stateVector, 61), get_channelName_if(channel_names, 62), get_state_if(stateVector, 62), get_channelName_if(channel_names, 63), get_state_if(stateVector, 63), get_channelName_if(channel_names, 64), get_state_if(stateVector, 64), get_channelName_if(channel_names, 65), get_state_if(stateVector, 65), get_channelName_if(channel_names, 66), get_state_if(stateVector, 66), get_channelName_if(channel_names, 67), get_state_if(stateVector, 67), get_channelName_if(channel_names, 68), get_state_if(stateVector, 68), get_channelName_if(channel_names, 69), get_state_if(stateVector, 69), get_channelName_if(channel_names, 70), get_state_if(stateVector, 70), get_channelName_if(channel_names, 71), get_state_if(stateVector, 71), get_channelName_if(channel_names, 72), get_state_if(stateVector, 72), get_channelName_if(channel_names, 73), get_state_if(stateVector, 73), get_channelName_if(channel_names, 74), get_state_if(stateVector, 74), get_channelName_if(channel_names, 75), get_state_if(stateVector, 75), get_channelName_if(channel_names, 76), get_state_if(stateVector, 76), get_channelName_if(channel_names, 77), get_state_if(stateVector, 77), get_channelName_if(channel_names, 78), get_state_if(stateVector, 78), get_channelName_if(channel_names, 79), get_state_if(stateVector, 79), get_channelName_if(channel_names, 80), get_state_if(stateVector, 80), get_channelName_if(channel_names, 81), get_state_if(stateVector, 81), get_channelName_if(channel_names, 82), get_state_if(stateVector, 82), get_channelName_if(channel_names, 83), get_state_if(stateVector, 83), get_channelName_if(channel_names, 84), get_state_if(stateVector, 84), get_channelName_if(channel_names, 85), get_state_if(stateVector, 85), get_channelName_if(channel_names, 86), get_state_if(stateVector, 86), get_channelName_if(channel_names, 87), get_state_if(stateVector, 87), get_channelName_if(channel_names, 88), get_state_if(stateVector, 88), get_channelName_if(channel_names, 89), get_state_if(stateVector, 89), get_channelName_if(channel_names, 90), get_state_if(stateVector, 90), get_channelName_if(channel_names, 91), get_state_if(stateVector, 91), get_channelName_if(channel_names, 92), get_state_if(stateVector, 92), get_channelName_if(channel_names, 93), get_state_if(stateVector, 93), get_channelName_if(channel_names, 94), get_state_if(stateVector, 94), get_channelName_if(channel_names, 95), get_state_if(stateVector, 95));
```

Comment:
I missed this last time: there needs to be 1 extra nullptr at the end to ensure that it's terminated when you pass in the max.

Reply:
oh, good catch! That means we need to pass one additional set of arguments.